src/components/Missions/MissionList.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import Table from 'react-bootstrap/Table';
+
+const MissionList = ({ missionsProp }) => (
+  <>
+    <Table striped bordered hover size="sm" style={{ marigin: '15px 25px' }}>
+      <thead>
+        <tr>
+          <th>Missision</th>
+          <th>Description</th>
+          <th>Status</th>
+          <th>#</th>
+        </tr>
+      </thead>
+      <tbody>
+        {
+      missionsProp.map((mission) => (
+        <tr key={mission.mission_id}>
+          <td>
+            {' '}
+            {mission.mission_name}
+            {' '}
+          </td>
+          <td>
+            {' '}
+            {mission.description}
+            {' '}
+          </td>
+          <td>
+            {' '}
+            <span className="status-text" style={{ background: '#d3d3d3' }}> Not a member </span>
+            {' '}
+          </td>
+          <td>
+            {' '}
+            <button type="button" style={{ borderRadius: '8px' }}> Join mission </button>
+            {' '}
+          </td>
```

Comment:
- [ Optional] Great job making the buttons, the style is ok for this stage, but after fetching the actual data, I would suggest you style each button according to the project requirements.